connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/WebhookControllerTestZeebeTests.java

```diff
@@ -184,7 +184,7 @@ public void testSuccessfulProcessingWithFailedActivation() throws Exception {
 
     var correlationHandlerMock = mock(InboundCorrelationHandler.class);
     when(correlationHandlerMock.correlate(any(), any()))
-        .thenReturn(new CorrelationResult.Failure.ActivationConditionNotMet());
+        .thenReturn(new CorrelationResult.Failure.ActivationConditionNotMet(false));
 
     var webhookDef = webhookDefinition("nonExistingProcess", 1, "myPath");
     var webhookContext =
```

Comment:
Can you also add a test case when the message acknowledgement is deactivated so the webhook should fail?

Reply:
Just to clarify, do you think we should always return an error response when activation condition is not met? Or should we also make this configurable, like for RabbitMQ / Kafka?
Maybe we could have a quick sync on Monday to discuss the desired behavior